### ISSUE TITLE
"All images" crash

### ISSUE THREAD

**Issue by mcginty** [WhisperSystems/TextSecure#3216]
```
05-21 00:56:01.855  15072-15072/org.thoughtcrime.securesms E/AndroidRuntime﹕ FATAL EXCEPTION: main
    Process: org.thoughtcrime.securesms, PID: 15072
    java.lang.IllegalArgumentException: column 'cid' does not exist
            at android.database.AbstractCursor.getColumnIndexOrThrow(AbstractCursor.java:303)
            at org.thoughtcrime.securesms.database.PartDatabase$ImageRecord.from(PartDatabase.java:555)
            at org.thoughtcrime.securesms.ImageMediaAdapter.onBindViewHolder(ImageMediaAdapter.java:69)
            at org.thoughtcrime.securesms.ImageMediaAdapter.onBindViewHolder(ImageMediaAdapter.java:41)
            at org.thoughtcrime.securesms.database.CursorRecyclerViewAdapter.onBindViewHolder(CursorRecyclerViewAdapter.java:103)
            at android.support.v7.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:4138)
            at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:3448)
            at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:3340)
            at android.support.v7.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:1810)
            at android.support.v7.widget.GridLayoutManager.layoutChunk(GridLayoutManager.java:356)
            at android.support.v7.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1269)
            at android.support.v7.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:523)
            at android.support.v7.widget.GridLayoutManager.onLayoutChildren(GridLayoutManager.java:151)
            at android.support.v7.widget.RecyclerView.dispatchLayout(RecyclerView.java:1988)
            at android.support.v7.widget.RecyclerView.onLayout(RecyclerView.java:2237)
            at android.view.View.layout(View.java:15678)
            at android.view.ViewGroup.layout(ViewGroup.java:5039)
            at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1077)
            at android.view.View.layout(View.java:15678)
            at android.view.ViewGroup.layout(ViewGroup.java:5039)
            at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
            at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
            at android.view.View.layout(View.java:15678)
            at android.view.ViewGroup.layout(ViewGroup.java:5039)
            at android.support.v7.internal.widget.ActionBarOverlayLayout.onLayout(ActionBarOverlayLayout.java:493)
            at android.view.View.layout(View.java:15678)
            at android.view.ViewGroup.layout(ViewGroup.java:5039)
            at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
            at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
            at android.view.View.layout(View.java:15678)
            at android.view.ViewGroup.layout(ViewGroup.java:5039)
            at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1703)
            at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1557)
            at android.widget.LinearLayout.onLayout(LinearLayout.java:1466)
            at android.view.View.layout(View.java:15678)
            at android.view.ViewGroup.layout(ViewGroup.java:5039)
            at android.widget.FrameLayout.layoutChildren(FrameLayout.java:579)
            at android.widget.FrameLayout.onLayout(FrameLayout.java:514)
            at android.view.View.layout(View.java:15678)
            at android.view.ViewGroup.layout(ViewGroup.java:5039)
            at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2086)
            at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1843)
            at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1061)
            at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:5891)
            at android.view.Choreographer$CallbackRecord.run(Choreographer.java:767)
            at android.view.Choreographer.doCallbacks(Choreographer.java:580)
            at android.view.Choreographer.doFrame(Choreographer.java:550)
            at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:753)
            at android.os.Handler.handleCallback(Handler.java:739)
            at android.os.Handler.dispatchMessage(Handler.java:95)
            at android.os.Looper.loop(Looper.java:135)
            at android.app.ActivityThread.main(ActivityThread.java:5293)
            at java.lang.reflect.Method.invoke(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:372)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:904)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:699)
```

**Comment by unrulygnu** [comment]
Sorry for the dup #3218 ; I checked then started pulling logs and writing my bug report just a few minutes before you posted this.

**Comment by 2-4601** [comment]
@unrulygnu Your bug report is so glorious that I think this issue should be closed as duplicate even though this was _first_.

**Comment by rhodey** [comment]
resolved by 1beed2103272dc61090c8c3eff54fedfbba93a3d
